SampleGetApi/Hooks/TestHooks.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using SampleGetApi.ReportsHelper;
+
+namespace SampleGetApi.Hooks
+{
+	public class TestHooks : IAsyncLifetime 
```

Comment:
What does this do?

Reply:
Will add a comment for the implementation. 
This get initialised at the start of the test Thread. Its more like setup and teardown functions